openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -710,6 +706,47 @@ Cache-Control: no-store
 }
 ```
 
+# Nonce Endpoint {#nonce-endpoint}
+
+This endpoint allows a Client to acquire a fresh `c_nonce` value without the overhead of a full Credential Request. A Credential Issuer that requires `c_nonce` values to be incorporated into proofs in the Credential Request (see (#credential-request)) MUST offer a Nonce Endpoint.
```

Comment:
What is the meaning of `fresh` in this context? A nonce that is valid (i.e. usable) at the time it was issued? Or does this require two distinct requests to provide two distinct nonces?

Reply:
yes, since a nonce is no more than once we assume that each request return an unique and fresh nonce value. I suppose that the term "fresh" made this implicit. I see space to replace it with something more explicit and with few words.

Reply:
discussed on the WG call
> fresh -> usable or valid at the time of issuance